### PR TITLE
upgrade to enterprise

### PR DESCRIPTION
**Pull request by nadinequinr3**

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Review by yarivus — APPROVED** [review]
LGTM

**Review by davidcurrie — DISMISSED** [review]
Parfait!

File: content/en/platform/corda/5.0/deploying-operating/deploying/enterprise.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{{< note >}}
+The configuration used for the Corda Enterprise installation should be the same as that for the original Corda Community installation. In particular, the configuration must grant access to the same database instance and Kafka cluster so that it can pick up the state from the previous cluster. It also must use the same salt and passphrase so that Corda can decrypt the configuration stored in the database.
+{{< /note >}}
+The following steps migrate an existing Corda Community deployment to Corda Enterprise, where the existing deployment has a Helm release name of `HELM_RELEASE_NAME` in the namespace `KUBERNETES_NAMESPACE`, installed with the overrides in the file `values.yaml`:
+
+{{< warning >}}
+The migration process results in downtime for the Corda cluster. 
+{{< /warning >}}
+
+1. Run the following to uninstall the Corda Community Helm release:
```
Comment (AnnaK-R3) [inline content/en/platform/corda/5.0/deploying-operating/deploying/enterprise.md:24]:
```suggestion
1. Uninstall the Corda Community Helm release:
```

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**